src/main/webapp/script.js

```diff
@@ -10,4 +10,25 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function openForm() {
+  document.getElementById("myForm").style.display = "block";
+  document.getElementById("popupButton").style.display = "none";
+}
+
+function closeForm() {
```

Comment:
Maybe change the names of openForm and closeForm to openImageForm and closeImageForm because once the image and profile code is merged together it might become somewhat difficult for someone reading the code for the first time to know which code goes with the image form or goes with the profile form.